src/timeline/Packer.js

```diff
@@ -0,0 +1,109 @@
+// @flow
+import moment from 'moment';
+const offset = 100;
```

Comment:
_A comment on the file._

I see that you use `const`, `let` and sometimes `var`.
can we please use ES6 only?

Reply:
Changed to ES6.